client/state/action-types.js

```diff
@@ -328,6 +328,7 @@ export const SEO_TITLE_SET = 'SEO_TITLE_SET';
 export const SIGNUP_DEPENDENCY_STORE_RESET = 'SIGNUP_DEPENDENCY_STORE_RESET';
 export const SIGNUP_DEPENDENCY_STORE_UPDATE = 'SIGNUP_DEPENDENCY_STORE_UPDATE';
 export const SIGNUP_STEPS_SITE_TITLE = 'SIGNUP_STEPS_SITE_TITLE';
+export const SIGNUP_STEPS_SURVEY = 'SIGNUP_STEPS_SURVEY';
```

Comment:
I think there's a precedent for `_SET`. We should probably go back and update `SIGNUP_STEPS_SITE_TITLE` too for consistency.